src/consensus/consensus.h

```diff
@@ -22,11 +22,15 @@
 
 #include <stdint.h>
 
+/** The maximum number of hashes allowed in the path from the auxiliary block
+ ** header to auxiliary block-final transaction.  Sufficient to support 2GB *
+ ** blocks on the auxiliary block chain. */
+static const unsigned int MAX_AUX_POW_BRANCH_LENGTH = 25;
 /** The maximum allowed size for a serialized block, in bytes (only for buffer size limits) */
 static const unsigned int MAX_BLOCK_SERIALIZED_SIZE = 4000000;
 /** The maximum allowed weight for a block, see BIP 141 (network rule) */
 static const unsigned int MAX_BLOCK_WEIGHT = 4000000;
-/** The maximum allowed size for a block excluding witness data, in bytes (network rule) */
+/** The maximum allowed size for a block excluding witness and auxiliary proof-of-work data, in bytes (network rule) */
```

Comment:
```suggestion
/** The maximum allowed size for a block excluding witness and auxiliary proof-of-work
 ** data, in bytes (network rule) */
```

Nit: you are capping comment lines above but not here.

Reply:
Fixed.